include/language/en_us.lang.php

```diff
@@ -2400,6 +2403,8 @@
     'LBL_DELETE_BUTTON_KEY' => 'd',
     'LBL_DELETE_BUTTON_LABEL' => 'Delete',
     'LBL_DELETE_BUTTON_TITLE' => 'Delete',
+    'LBL_BULK_ACTION_BUTTON_LABEL' => 'BULK ACTION',
```

Comment:
@horus68 This label "bulk action" is intentional. You can translate it correctly in other languages where the capital letters denote some kind of meaning. Though it raises an issue with the CSS for the create and action buttons. So we either need to have it customisation in the labels or forced in the CSS. Forcing in the CSS means that you will have no control over the capitalisation in your translations.


Example:
![bulk_action](https://cloud.githubusercontent.com/assets/12231216/21399195/96abe256-c7a2-11e6-8ebf-462070d63406.png)